package.json

```diff
@@ -13,37 +13,40 @@
   "devDependencies": {
     "autoprefixer-core": "5.2.1",
     "babel-core": "6.5.2",
-    "babel-loader": "^6.2.4",
+    "babel-loader": "6.2.4",
     "babel-preset-es2015": "6.5.0",
-    "babel-register": "^6.7.2",
-    "chai": "^3.5.0",
-    "css-loader": "^0.12.0",
-    "eslint": "^2.11.1",
-    "eslint-config-rambler": "^0.1.0",
-    "eslint-loader": "^1.3.0",
-    "extract-text-webpack-plugin": "^0.7.0",
-    "file-loader": "^0.8.1",
-    "glob": "^7.0.3",
-    "html-webpack-plugin": "^1.3.0",
-    "jade": "^1.11.0",
+    "babel-register": "6.7.2",
+    "chai": "3.5.0",
+    "css-loader": "0.12.0",
+    "eslint": "2.11.1",
+    "eslint-config-rambler": "0.1.0",
+    "eslint-loader": "1.3.0",
+    "extract-text-webpack-plugin": "0.7.0",
+    "file-loader": "0.8.1",
+    "glob": "7.0.3",
+    "html-webpack-plugin": "1.3.0",
+    "jade": "1.11.0",
     "jade-loader": "0.7.1",
-    "json-loader": "^0.5.1",
-    "lodash": "^3.7.0",
-    "manifest-revision-webpack-plugin": "^0.1.0",
-    "mocha": "^2.4.5",
-    "npmlog": "^2.0.3",
-    "path": "^0.11.14",
-    "phantomjs-prebuilt": "^2.1.4",
-    "postcss-loader": "^0.6.0",
-    "raw-loader": "^0.5.1",
-    "style-loader": "^0.12.1",
-    "stylint": "^1.3.10",
-    "stylint-loader": "^1.0.0",
-    "stylus-loader": "^1.1.1",
-    "testem": "^1.6.0",
+    "json-loader": "0.5.1",
+    "lodash": "3.7.0",
+    "manifest-revision-webpack-plugin": "0.1.0",
+    "mocha": "2.4.5",
+    "npmlog": "2.0.3",
+    "path": "0.11.14",
+    "phantomjs-prebuilt": "2.1.4",
+    "postcss-loader": "0.6.0",
+    "raw-loader": "0.5.1",
+    "style-loader": "0.12.1",
+    "stylint": "1.3.10",
+    "stylint-loader": "1.0.0",
+    "stylus-loader": "1.1.1",
```

Comment:
@lgordey 

⇒  npm shrinkwrap -dev
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.10.0/bin/node" "/usr/local/bin/npm" "shrinkwrap" "-dev"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.3

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! missing: babel-register@^6.9.0, required by babel-core@6.10.4
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ianbrode/Documents/dev/js/invaders/npm-debug.log